### ISSUE TITLE
Large MonoBehaviour file losing a lot of data, update required

### ISSUE THREAD

**Issue by A-nonymous027** [DerPopo/UABE#405]
I am currently modding an app using two assets opener, AssetStudio (but cannot import) and UABE.
In AssetStudio, I stumbled across one particular MonoBehaviour file I want to modify, it looks like this:
![image](https://user-images.githubusercontent.com/79244637/108314577-b0f50400-71ec-11eb-98a7-7f9c26deca26.png)

But when I opened **that exact same file** in UABE, I only got this:
![image](https://user-images.githubusercontent.com/79244637/108315407-f534d400-71ed-11eb-824e-31dd58028c5c.png)

This is bad. The file is as large as 1752 bytes but UABE can only read the first 32 bytes.
**Both AssetStudio and UABE have been provided the same assembly files to get script information.**

Is there anything I can do about this? If not, please fix this.

**Comment by webfischi** [comment]
No, I'm having the same issues, UABE always wants to have some weird .dll files, that don't exist, while AssetStudio is happy with the Managed folder, and I think that is the main difference. I would like it more, if UABE could import dumps exported by AssetStudio, that would already circumvent a lot of headaches.
